lib/presentation/app.dart

```diff
@@ -30,8 +32,13 @@ class _AppState extends ConsumerState<App> {
         return MaterialApp(
           // debugShowCheckedModeBanner: false,
           theme: Themes.appTheme(),
-          // localizationsDelegates: AppLocalizations.localizationsDelegates,
-          // supportedLocales: AppLocalizations.supportedLocales,
+          localizationsDelegates: const [
```

Comment:
i agree with you but is there any problem i used same code as link?
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/14831652/202603736-73e4e76d-ca1b-47e8-9d94-90f2ce16ccdc.png">

Reply:
I guess no problem, since both works, but if there is an option to use a cleaner definition then should we not use it?
I used the abbreviated version in all previous projects without issue : 
```
  localizationsDelegates: AppLocalizations.localizationsDelegates,
  supportedLocales: AppLocalizations.supportedLocales,
```

Reply:
https://stackoverflow.com/questions/70066318/flutterios-no-cupertinolocalizations-found-how-fix-it

Reply:
@diarmaidlindsay thank you have got good point
basically generated code have above those lines what i explained.
so i will use the lines were before.
<img width="948" alt="image" src="https://user-images.githubusercontent.com/14831652/202944906-14880d73-7f6f-4267-bb5a-fd1e7c7e2645.png">